aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteCommandFactory.java

```diff
@@ -47,6 +47,7 @@
 import java.util.List;
 import java.util.Objects;
 import java.util.Set;
+import java.util.stream.Collectors;
```

Comment:
We have talked about using Java 8 Stream APIs before, and decided against it. 

It looks like you're using it to join items by a comma `,`. For that, please use Android's [`TextUtils.join(CharSequence, Iterable)`](https://developer.android.com/reference/android/text/TextUtils#join(java.lang.CharSequence,%20java.lang.Iterable)):

```java
String fooBar = TextUtils.join(", ", Arrays.asList("foo", "bar"))
```

Reply:
Haha that's the utility I was looking for. Didn't know that Android provides it.